templates/ratecard-rates.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a apigee rate plan detail.
+ *
+ * Variables:
+ * - detail:      See: "rate-plan-detail.html.twig"
+ * - rates:
+ *   - id:                          The unique ID of the rate plan rate.
+ *   - startUnit:                   The number the rate starts at.
+ *   - endUnit:                     The rate is good until this nuber of units.
+ *   - rate:                        The rate that is applies within this range.
+ */
+#}
+
+<div class="rate_plan__detail-card__ratecard-rates rate-list-wrapper">
```

Comment:
Maybe something like `<div class="rate-plan-rates">`.  And really this template and the one below should just either be in the base hook template, or they should have their own theme hook.  Using `rate_plan_detail` as a base hook for this is odd. 

Then you can add another class, a modifier, so signify the type, e.g. `rate-plan-rates--ratecard` and in the template below `rate-plan-rates--revshare`.

Reply:
Cool. done.